### PR TITLE
Add harper extension

### PR DESCRIPTION
**Pull request by Stef16Robbe**

### CODE REVIEW

**Comment by Stef16Robbe** [comment]
@cla-bot check

**Comment by Stef16Robbe** [comment]
Totally missed that CI failed on the extension name, should be fixed now ^^

File: extensions.toml [hunk 1]
```diff
@@ -510,6 +510,10 @@ version = "1.0.0"
 submodule = "extensions/halcyon"
 version = "0.0.5"
 
+[harper-ls]
```
Comment (maxdeviant) [inline extensions.toml:513]: Could we change the extension ID to just `harper`?

Comment (maxdeviant) [reply extensions.toml]: > Does that mean I should init and update all submodules on my fork?

If you see an error from other extensions on your branch you typically just need to merge in `main` (no need to do anything with the other submodules).

**Pull request merged**